### ISSUE TITLE
Error during trash operation: system cannot find the file specified

### ISSUE THREAD

**Issue by qu4ntumrush** [qarmin/czkawka#1353]
**Bug Description**
Cannot delete files found during Similar Image task

**Steps to reproduce:**

Previously was able to delete without problem, and I could clearly find and delete the file in Explorer

**Log**
Failed to delete file H:\X.jpg ⁨reason ⁨Error during a `trash` operation: Os { code: -2147024894, description: "windows error: The system cannot find the file specified. (0x80070002)"}

**System**

- Czkawka/Krokiet version: 7.0.0 cli/gui
- OS version: Windows 10 64-bit
- Installation method: github binary

**Comment by KaedeInSpace** [comment]
> Cannot delete files found during Similar Image task located on a NAS drive
What filesystem is the NAS drive of? Because if it's a common linux one, then likely this could be a case sensitivity issue.

I have not run into this issue myself, but have noticed that the Krokiet GUI app seems to change the case on me (to lower) in the logs and when having the GUI open directories. Though, I only use windows filesystems so far for this app.

**Comment by qu4ntumrush** [comment]
Yes, it's formatted to EXT4. It used to be able to delete EXT4 files. I see the nightly build was just released but I use it on Windows.

**Comment by hydr0homiee** [comment]
@Mewyk I'm running into this same issue on a Synology NAS using BTFRS. I'm using the windows_czkawka_412 release for 9.0.0 on Windows 10 and have the NAS connected as a network drive.

Would it be possible to fix this if it is a case sensitivity issue, or would I need to find an alternate tool or manually delete every file?

**Comment by Mewyk** [comment]
Sorry, but I am not really sure. I actually ended up not using this program anymore (because I didn't need a GUI anymore) as I am just writing in my own duplicate check stuff to my organizational app I have.

Hopefully someone else can help you!

**Comment by hydr0homiee** [comment]
Interestingly enough, after doing some more research I tried turning off `Move deleted file to trash` and that seemed to fix it. I'm guessing it's an issue with whatever flags are being used when that option is toggled on being incompatible with Windows.

Appreciate the quick response nonetheless!

**Comment by oda112** [comment]
> Interestingly enough, after doing some more research I tried turning off `Move deleted file to trash` and that seemed to fix it. I'm guessing it's an issue with whatever flags are being used when that option is toggled on being incompatible with Windows.
> 
> Appreciate the quick response nonetheless!

This worked on my setup as well (NTFS NAS over LAN)

**Comment by qarmin** [comment]
The new (not yet released) version includes two buttons: one for moving files to the trash and another for permanently deleting them (instead of a single button with behavior configurable via a settings option).

Some network drives, as well as certain local Linux drives mounted with specific configurations(like mine), do not support the trash feature. In such cases, even the file manager refuses to move files to the trash. Because of this, the operation is in popup marked as potentially fallible.
